rustls/src/crypto/ring/mod.rs

```diff
@@ -44,6 +30,38 @@ impl CryptoProvider for Ring {
     }
 }
 
+// Modules below here should always use `super::ringlike` to access a
```

Comment:
Why is this order better? I feel like we usually keep module declarations and module-level imports before any other items.

Reply:
have dropped this commit, but please re-review the ordering in `src/crypto/{ring,aws-lc-rs}/mod.rs`